Console/UI/Pages/Status.xaml.cs

```diff
@@ -53,33 +57,67 @@ private void btnSave_Click(object sender, RoutedEventArgs e)
             }
             status.Save();
 
-            if (!isInstalled &&
-                ((status.PrivateIsEnabled && status.PrivateIsOutBlockedNotif)
-                || (status.PublicIsEnabled && status.PublicIsOutBlockedNotif)
-                || (status.DomainIsEnabled && status.DomainIsOutBlockedNotif)))
+            bool checkResult(Func<bool> boolFunction, string okMsg, string errorMsg)

```

Comment:
Shouldn't it be handled by the InstallHelper (which will be renamed soon ;-) ), keeping a callback on the UI side? I guess it's what I did (don't remember exactly), but this is what I'd do anyway. Since the logging can be handled in InstallHelper as well, only LastMessage affectation is useful here and it could be done by said callback.

Reply:
Well, a similar callback was there already, just made the callback handle the common parts which were repeated in the the helper. Could be done better I agree.

Reply:
You can keep the exact same logic, while keeping the callback for the LastMessage. Anyway this won't block the merge, we can have a look at this later.